src/permissions/PermissionController.ts

```diff
@@ -845,6 +860,24 @@ export class PermissionController<
     return Boolean(this.getPermission(origin, target));
   }
 
+  /**
+   * Checks whether the subject with the specified origin has all the specified
+   * permissions.
+   *
+   * @param origin - The origin of the subject.
+   * @param targets - The target names of every permission.
+   * @returns Whether the subject has all the permissions.
```

Comment:
Nit: This sounds like it's suggesting that `targets` _must_ be a list of all permissions. But this is just the list of permissions being checked, right? And similarly the return description could use the same term as the method description.

```suggestion
   * @param targets - The list of permissions to check.
   * @returns Whether the subject has all the specified permissions.
```